gitenberg/push.py

```diff
@@ -98,14 +98,13 @@ def format_desc(self):
 
     def create_repo(self):
         try:
-            self.repo = self.org.create_repo(
+            self.repo = self.org.create_repository(
                 self.book.repo_name,
                 description=self.format_desc(),
                 homepage=self.org_homepage,
                 private=False,
                 has_issues=True,
-                has_wiki=False,
-                has_downloads=True
```

Comment:
why unset has_downloads? is it not needed anymore?

Reply:
It doesn't exist any more: https://blog.github.com/2012-12-12-goodbye-uploads/. I can't find an equivalent has_releases too, but it looks like the new repo has releases enabled just as much as any of the others which don't have any yet.

Reply:
odd! thanks for finding this; probably enforced by the github3 update

Reply:
No problem. What actually happened is I hit the problem without touching anything because setup.py specifies >=, so pip/setuptools grabbed a newer version, and then I updated the dependency to ensure people upgrading will get a new enough version that the new name works.